.github/actions/setup-rust/action.yml

```diff
@@ -4,7 +4,8 @@ description: "Toolchain setup and Initial compilation"
 inputs:
   repo-token:
     description: "GitHub token for accessing the repository (typically secrets.GITHUB_TOKEN)"
-    required: true
+    required: false
+    default: ""
```

Comment:
why not default to `${{ secrets.GITHUB_TOKEN }}`?

Reply:
Also this is just to avoid rate limiting.

Reply:
so shouldn't we set it to that?

Reply:
Its not defined? https://github.com/vortex-data/vortex/pull/5001#discussion_r2444840420